### PR TITLE
Updated the translation for Simplified Chinese.

### PR DESCRIPTION
**Pull request by 2805671972**
Updated the translation for Simplified Chinese, fully aligning it with the latest content.

## Additions

Describe new functionality added by your code, e.g.
- Tribal smoke bombs
- New tribal smoke bomb sprite
- Tribal smoke bomb recipes at smithing bench and crafting spot using prometheum

## Changes

Describe adjustments to existing features made in this merge, e.g.
- Increased regular smoke bomb radius

## References

Links to the associated issues or other related pull requests, e.g.
- Closes #[ISSUE_NUMBER]
- Contributes towards #[ISSUE_NUMBER]

## Reasoning

Why did you choose to implement things this way, e.g.
- Tribals need ways to close distance with pirate raiders
- Smoke bombs allow this while enhancing combat micro
- Thematically appropriate as we already allow tribal prometheum handling
- Easy to implement
- Buffed regular smoke grenades as they are rarely utilized and to justify additional investment

## Alternatives

Describe alternative implementations you have considered, e.g.
- Tribal catapult that launches melee animals into siege camps:
  - Additional use for animals
  - Anachronistic
  - Breaks realism theme

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [ ] (For compatibility patches) ...with and without patched mod loaded
- [x] Playtested a colony (specify how long)

### CODE REVIEW

File: Languages/ChineseSimplified/DefInjected/CombatExtended.AmmoSetDef/Mech.xml [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
-
-	<AmmoSet_MechCharged.label>机械族电荷弹</AmmoSet_MechCharged.label>
-	<AmmoSet_MechShell.label>机械族炮弹</AmmoSet_MechShell.label>
-
+  
+  <!-- EN: mech charged ammo -->
+  <AmmoSet_MechCharged.label>机械族电荷弹</AmmoSet_MechCharged.label>
+  
+  <!-- EN: mech shell (Demo) -->
+  <AmmoSet_MechShell_Demo.label>机械族炮弹(演示)</AmmoSet_MechShell_Demo.label>
```
Comment (sumghai) [inline Languages/ChineseSimplified/DefInjected/CombatExtended.AmmoSetDef/Mech.xml:8]: Demo here is supposed to mean demolition (爆破), **_not_** demonstration (演示)

File: Languages/ChineseSimplified/DefInjected/RecipeDef/Mech.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  
+  <!-- EN: make Mech Charged cartridge x500 -->
+  <MakeAmmo_Mech_Charged.label>制作机械族电荷弹 x200</MakeAmmo_Mech_Charged.label>
+  <!-- EN: Craft 500 Mech Charged cartridges. -->
+  <MakeAmmo_Mech_Charged.description>制作机械族电荷弹 x200。</MakeAmmo_Mech_Charged.description>
+  <!-- EN: Making Mech Charged cartridges. -->
+  <MakeAmmo_Mech_Charged.jobString>正在制作机械族电荷弹</MakeAmmo_Mech_Charged.jobString>
+  
+  <!-- EN: make Mech Shell (Demo) x5 -->
+  <MakeAmmo_MechShell_Demo.label>制作机械族炮弹 (演示) x5</MakeAmmo_MechShell_Demo.label>
```
Comment (sumghai) [inline Languages/ChineseSimplified/DefInjected/RecipeDef/Mech.xml:12]: Demo here is supposed to mean demolition (毁灭), **_not_** demonstration (演示)

File: Languages/ChineseSimplified/DefInjected/RecipeDef/Mech.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  <!-- EN: Craft 5 Mech Shells (Demo). -->
```
Comment (sumghai) [inline Languages/ChineseSimplified/DefInjected/RecipeDef/Mech.xml:13]: Demo here is supposed to mean demolition (毁灭), **_not_** demonstration (演示)

File: Languages/ChineseSimplified/DefInjected/RecipeDef/Mech.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  <MakeAmmo_MechShell_Demo.description>制作机械族炮弹 (演示) x5。</MakeAmmo_MechShell_Demo.description>
```
Comment (sumghai) [inline Languages/ChineseSimplified/DefInjected/RecipeDef/Mech.xml:14]: Demo here is supposed to mean demolition (毁灭), **_not_** demonstration (演示)

File: Languages/ChineseSimplified/DefInjected/RecipeDef/Mech.xml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  <!-- EN: Making Mech Shells (Demo). -->
+  <MakeAmmo_MechShell_Demo.jobString>正在制作机械族炮弹 (演示)</MakeAmmo_MechShell_Demo.jobString>
```
Comment (sumghai) [inline Languages/ChineseSimplified/DefInjected/RecipeDef/Mech.xml:16]: Demo here is supposed to mean demolition (毁灭), **_not_** demonstration (演示)

File: Languages/ChineseSimplified/DefInjected/ThingDef/5x50mmCaseless.xml [hunk 6]
```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<LanguageData>
+  
+  <!-- EN: 5x50mm Caseless (Sabot) -->
+  <Ammo_5x50mmCaseless_Sabot.label>5x50mm 无壳弹 (脱壳穿甲弹)</Ammo_5x50mmCaseless_Sabot.label>
+  <!-- EN: A long caseless cartridge with a solid propellant enveloping the slim projectile.
+\n\n脱壳穿甲弹:
```
Comment (sumghai) [inline Languages/ChineseSimplified/DefInjected/ThingDef/5x50mmCaseless.xml:7]: The Simplified Mandarin text in the comments should be removed.

File: Languages/ChineseSimplified/DefInjected/ThingDef/Mech.xml [hunk 7]
```diff
@@ -1,14 +1,19 @@
...
-	<Ammo_MechShell.description>机械族火炮和重型武器使用的大口径炮弹。</Ammo_MechShell.description>
-
+  
+  <!-- EN: Mech Charged cartridge -->
+  <Ammo_Mech_Charged.label>机械族电荷弹</Ammo_Mech_Charged.label>
+  <!-- EN: Charged shot ammo used by mechanoid weaponry. -->
+  <Ammo_Mech_Charged.description>用于机械族武器的电荷弹。</Ammo_Mech_Charged.description>
+  
+  <!-- EN: mech shell (Demo) -->
+  <Ammo_MechShell_Demo.label>机械族炮弹 (演示)</Ammo_MechShell_Demo.label>
```
Comment (sumghai) [inline Languages/ChineseSimplified/DefInjected/ThingDef/Mech.xml:10]: Demo here is supposed to mean demolition (破拆), **_not_** demonstration (演示)

**Review by sumghai — CHANGES_REQUESTED** [review]
@2805671972 - Most of the translation updates look good, but please fix the translations for the mechanoid demo(lition) ammo - "demo" in this context is supposed to mean demolition (爆破 or 破拆), **_not_** demonstration (演示)

File: Languages/ChineseSimplified/DefInjected/ThingDef/Mech.xml (hunk 7 above)
Comment (2805671972) [reply Languages/ChineseSimplified/DefInjected/ThingDef/Mech.xml]: The translation content has been revised

File: Languages/ChineseSimplified/DefInjected/CombatExtended.AmmoSetDef/Mech.xml (hunk 1 above)
Comment (sumghai) [reply Languages/ChineseSimplified/DefInjected/CombatExtended.AmmoSetDef/Mech.xml]: Fixed by @2805671972 in 1343cac86db6aa7a1cafe99a467e5bd37f999f61

File: Languages/ChineseSimplified/DefInjected/RecipeDef/Mech.xml (hunk 2 above)
Comment (sumghai) [reply Languages/ChineseSimplified/DefInjected/RecipeDef/Mech.xml]: Fixed by @2805671972 in 1343cac86db6aa7a1cafe99a467e5bd37f999f61

File: Languages/ChineseSimplified/DefInjected/RecipeDef/Mech.xml (hunk 4 above)
Comment (sumghai) [reply Languages/ChineseSimplified/DefInjected/RecipeDef/Mech.xml]: Fixed by @2805671972 in 1343cac86db6aa7a1cafe99a467e5bd37f999f61

File: Languages/ChineseSimplified/DefInjected/RecipeDef/Mech.xml (hunk 5 above)
Comment (sumghai) [reply Languages/ChineseSimplified/DefInjected/RecipeDef/Mech.xml]: Fixed by @2805671972 in 1343cac86db6aa7a1cafe99a467e5bd37f999f61

File: Languages/ChineseSimplified/DefInjected/ThingDef/5x50mmCaseless.xml (hunk 6 above)
Comment (sumghai) [reply Languages/ChineseSimplified/DefInjected/ThingDef/5x50mmCaseless.xml:7]: I'll fix this myself later.

File: Languages/ChineseSimplified/DefInjected/ThingDef/Mech.xml (hunk 7 above)
Comment (sumghai) [reply Languages/ChineseSimplified/DefInjected/ThingDef/Mech.xml]: Fixed by @2805671972 in 1343cac86db6aa7a1cafe99a467e5bd37f999f61

**Review by sumghai — APPROVED** [review]

**Pull request merged**